src/simulation/dynamics/DynOutput/boreAngCalc/boreAngCalc.h

```diff
@@ -42,25 +42,29 @@ class BoreAngCalc: public SysModel {
     
     void Reset(uint64_t CurrentSimNanos);
     void UpdateState(uint64_t CurrentSimNanos);
-    void computeAxisPoint();
-    void computeOutputData();
+    void computeCelestialAxisPoint();
+    void computeCelestialOutputData();
+    void computeInertialOutputData();
     void WriteOutputMessages(uint64_t CurrentClock);
     void ReadInputs();
     
-public:
     ReadFunctor<SCStatesMsgPayload> scStateInMsg;       //!< (-) spacecraft state input message
     ReadFunctor<SpicePlanetStateMsgPayload> celBodyInMsg;   //!< (-) celestial body state msg at which we pointing at
     Message<BoreAngleMsgPayload> angOutMsg;                 //!< (-) bore sight output message
 
-    BoreAngleMsgPayload boresightAng; //!< (-) Boresight angles relative to target
-    bool inputsGood;                  //!< (-) Flag indicating that inputs were read correctly
-    BSKLogger bskLogger;                      //!< -- BSK Logging
     Eigen::Vector3d boreVec_B;                //!< (-) boresight vector in structure
     Eigen::Vector3d boreVec_Po;             //!< (-) pointing vector in the target relative point frame
+    Eigen::Vector3d inertialHeadingVec_N;     //!< (-) inertial boresight vector
 
 private:
     SpicePlanetStateMsgPayload localPlanet;//!< (-) planet that we are pointing at
     SCStatesMsgPayload localState;   //!< (-) observed state of the spacecraft
+
+    BoreAngleMsgPayload boresightAng = {};   //!< (-) Boresight angles relative to target
```

Comment:
:small_red_triangle_down: These are good refactorings. Important that we clean up as we're passing through the 'neighborhood'. However, they do not relate to the goal of this commit and should be pulled out into a separate commit. Similarly for the removal of the redundant return; statement in the constructor.

Reply:
Makes sense. I've separated this commit and created a new one that does this generic refactoring.